README.md

```diff
@@ -93,7 +93,7 @@ To run `[[Resolve]](promise, x)`, perform the following steps:
       1. If/when `rejectPromise` is called with a reason `r`, reject `promise` with `r`.
       1. If both `resolvePromise` and `rejectPromise` are called, or multiple calls to the same argument are made, the first call takes precedence, and any further calls are ignored.
       1. If calling `then` throws an exception `e`,
-         1. If `resolvePromise` or `rejectPromise` have been called, ignore it.
+         1. If `resolvePromise` or `rejectPromise` have been called, ignore the exception `e`.
```

Comment:
When I read "it", I wrongly thought "it" refers to either `resolvePromise` or `rejectPromise`. This change should prevent the next possible confusion.

Reply:
Yeah, I think it's perfectly reasonable to change "it" to "`e`" here.